lib/Bindings/Python/circt/support.py

```diff
@@ -348,6 +348,10 @@ def __getattr__(self, name):
       value = self.opview.results[index]
       return OpOperand(self.opview.operation, index, value, self)
 
+    # Forward "attributes" attribute from the operation.
+    if name == "attributes":
+      return self.opview.operation.attributes
```

Comment:
This makes it possible to directly access operation's attributes through NamedValueOpView, e.g. `cmpreg.attributes["sv.attributes"] = sv_attr`. Does this change make sense to you @mikeurbach?